Formula/c/cadence-workflow.rb

```diff
@@ -7,6 +7,11 @@ class CadenceWorkflow < Formula
   license "MIT"
   head "https://github.com/uber/cadence.git", branch: "master"
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
  # labeled as "pre-release" on GitHub before the version is released, so it's
  # necessary to use the `GithubLatest` strategy.
  livecheck do
```

The `stable` URL isn't a GitHub release asset, so we should explain why we're using the `GithubLatest` strategy.